dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -413,6 +413,8 @@
 			health-test-config = <0xa2b3>;
 			status = "disabled";
 			label = "RNG";
+			health-test-magic = <0xf00d00>;
```

Comment:
From the value I guess that should be `nist-config` instead of `health-test-magic`